offline_traces.md

```diff
@@ -0,0 +1,18 @@
+### What Are Offline Traces? 
+
+You can export traces and view them later using the Offline Traces page. 
+
+**Option 1**:  
+
+1. Export traces via the Traces Browser. 
+1. Save them locally as JSON files. 
+1. View them later using the Offline Traces page. 
+
+**Option 2**: 
+
+1. Export traces via the API. 
```

Comment:
Link to the API page on VMware code? Let's not do it if it's fussy.

Reply:
Since docs have more details on it and a screenshot of the specific API, I think let's not link for now. It's hard to get a URL to the specific API.